core/src/main/java/com/cloudinary/android/MediaManager.java

```diff
@@ -232,13 +244,6 @@ public Cloudinary getCloudinary() {
      */
     public Url url() {
         Url url = cloudinary.url();
-
-        // set https as default for android P and up - in P the default policy fails all http
-        // requests
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
```

Comment:
why was this removed in this PR?

Reply:
It was not removed, it was moved to the MediaManager `init` function instead of the `URL` init

Reply:
See my comment above, now it needs to be removed:)